### DISCUSSION TITLE
Adding "triggered by OpenHasp Script- From Plate 1" In  HA Logbook

### DISCUSSION THREAD

**Discussion by DanBendavid** [HASwitchPlate/openHASP#377] [Ideas]
I am experiencing ghost switching of all my lights, i am suspecting a OpenHasp plate to trigger all lights "on" in the middle of the night, this is happening when backlight and moodlight are switching On/Off .

Changing the state of a switch from a OpenHasp Plate must be recorded in HA Logbook with more information than it is in OpenHasp 0.6.3.

**Comment by fvanroie** [comment]
Do you have [debugging](https://www.openhasp.com/0.6.3/integrations/home-assistant/howto/#debugging) enabled for the openHASP Custom Component in HA?
